### ISSUE TITLE
Feature request: Custom validator to check if the field is not empty.

### ISSUE THREAD

**Issue by mdehaan** [Jako/AjaxUpload#22]
Custom validator to check if the field is not empty.
This almost works: Afbeeldingen_fileid:required` .
If the other to be validated fields are correct on first submission, then it works like a charm.
But after the form is submitted and an other to be validated field was left empty, then if you re-submit after correction, then it doesn't work anymore and will not validate the ajaxupload field.

**Comment by travisbotello** [comment]
Here's a simple workaround for the moment. Create a snippet and attach it to FormIt `&hooks`:

```
<?php
$image = $hook->getValue('image'); 
$success = false;

if(!empty($image)) {
    $success = true;
} else {
    $errorMsg = 'Please upload an image.';
    $hook->addError('image',$errorMsg);
}

return $success;
```

**Comment by Jako** [comment]
This could be added to AjaxUpload2Formit with an ajaxuploadRequired and ajaxuploadRequiredMessage property.

**Comment by CamilleBeaute** [comment]
I have a need for this as well! It would be great if the ajaxuploadRequired and ajaxuploadRequiredMessage properties mentioned above could be added!

**Comment by Jako** [comment]
This is available with 1.6.2.
